src/OpenColorIO/Baker.cpp

```diff
@@ -213,7 +213,14 @@ OCIO_NAMESPACE_ENTER
             err << "' could not be found. ";
             throw Exception(err.str().c_str());
         }
-        
+
+        if(!getConfig())
+        {
+            std::ostringstream err;
+            err << "No OCIO config has been set";
+            throw Exception(err.str().c_str());
```

Comment:
These three lines could be merged in a single line like:
`throw Exception("No OCIO config has been set.");`

Reply:
Merged.